doc/content/getting-started/migrating/migrating-from-v2/_index.md

```diff
@@ -0,0 +1,90 @@
+<br>
+</br>
+- The **Rx1 Delay** value for your end device needs to be set to `5 seconds`, which is a default for {{% tts %}} ({{% ttnv2 %}} is using `1 second`).
+    
+    OTAA devices migrated without their active session will negotiate about this value with {{% tts %}}, so it will be automatically set to `5 seconds` without your intervention.
+    
+    However, this needs to be manually set for OTAA devices migrated with an active session by [configuring their MAC settings]({{< ref "/getting-started/migrating/configure-mac-settings" >}}). It is also possible to re-program ABP devices to enforce the **Rx1 Delay** of `5 seconds` (this basically implies starting a new ABP session, i.e. migrating ABP devices without their active session from {{% ttnv2 %}}).
+
+    Otherwise, if **Rx1 Delay** keeps the default value of `1 second` from {{% ttnv2 %}}, the traffic routed by the Packet Broker will probably never reach {{% tts %}} in time, and the only solution is to migrate your gateway to {{% tts %}} too. Even if you migrate your gateway to {{% tts %}}, if it is using a high-latency backhaul, you might still experience issues.
+
+If these conditions are not met, you will most probably experience difficulties in communication between your end device and {{% tts %}} via Packet Broker. If you are using deployments that are not connected to Packet Broker, or you just want to migrate your devices with their active sessions but you are not migrating specifically from The Things Industries V2 to {{% tts %}} Cloud, you will have to [migrate your gateway]({{< ref "/getting-started/migrating/gateway-migration" >}}) as well in order to receive the traffic from your end device on {{% tts %}}.
+
+{{< warning >}} Theoretically, it is possible to migrate active device sessions from The Things Network V2 (public community network) to many {{% tts %}} deployments. But, this is achievable only if you [migrate your gateway to {{% tts %}}]({{< ref "/getting-started/migrating/gateway-migration" >}}), as the Packet Broker cannot route this traffic properly. 
+
+However, we strongly recommend The Things Network community members to keep their gateways registered on The Things Network V2 for as long as possible, or to agree on performing coordinated migration to {{% tts %}}, as this will ensure they will not loose their LoRaWAN coverage. We also strongly recommend to migrate devices without their active sessions, to make sure the Packet Broker can route the traffic properly and that it reaches {{% tts %}} in time.
```

Comment:
What is the audience for this page: The Things Network community or The Things Industries V2 SaaS customers? Should we have separate sections/tabs/pages? There's a lot of text on this page and the more people read things that are not intended for them, the sooner people stop reading and try stupid things

Reply:
The audience is everyone. This is a generic page, so TTN users have to be able to follow it too, even though they have their own dedicated, simplified documentation on TTN.org page.

We can try separating the info into multiple pages, but that will introduce even more repetition. Maybe @laurensslats has an idea how we can improve this?